Meta/Lagom/Tools/CodeGenerators/LibUnicode/GenerateUnicodeData.cpp

```diff
@@ -133,6 +144,24 @@ struct UnicodeData {
     PropList sentence_break_props;
 };
 
+static void sanitize_entry(StringView& entry)
+{
+    entry = entry.replace("-", "_", true);
+    entry = entry.replace(" ", "_", true);
```

Comment:
This is causing use-after-free - `entry` is a `StringView`, but `replace` returns a `String`, so we have a view into an immediate freed string here.
https://dev.azure.com/SerenityOS/SerenityOS/_build/results?buildId=10386&view=logs&jobId=2351e412-c793-598f-27f8-72c4f2d55ba1&j=2351e412-c793-598f-27f8-72c4f2d55ba1&t=ee375dfa-6b7d-514a-7789-3948d04c0dd3